### ISSUE TITLE
Selecting different samplers?

### ISSUE THREAD

**Issue by jmiskovic** [bes-dev/stable_diffusion.openvino#24]
Some of SD interfaces expose the ability to select the sampler. In particular, the `k_euler_a` is known to produce good results with smaller number of iterations, which would be fantastic for CPU version.

Where does this framework select the sampler implementation? Could it be exposed in the shell arguments?

**Comment by bes-dev** [comment]
@jmiskovic no, we do not provide functionality to select different samplers

**Comment by Drake53** [comment]
I found that DDIM is also good for fast results (3~5 iterations), though you can't really use it as a preview because the image usually changes a lot with more iterations.

I added support for DDIM in my fork: https://github.com/Drake53/stable_diffusion.openvino/commit/a56987c6bb1b8500e8662f3666d042bbdbd5296c
Had to set offset to 0, otherwise it would do 1 iteration less than I asked for.
